modules/textures/src/compressed-texture-loader.js

```diff
@@ -20,13 +20,12 @@ export const CompressedTextureWorkerLoader = {
     'dds', // WEBGL_compressed_texture_s3tc, WEBGL_compressed_texture_atc
     'pvr' // WEBGL_compressed_texture_pvrtc
   ],
-  mimeTypes: ['application/octet-stream'],
-  // tests: [new Uint8Array(PVR_MAGIC_BYTES).buffer],
+  mimeTypes: ['application/octet-stream', 'image/vnd-ms.dds'],
```

Comment:
From https://www.iana.org/assignments/media-types/media-types.xhtml, looks like ktx has two extensions and two mimetypes.

```
ktx | image/ktx | [Khronos][Mark_Callow]
ktx2 | image/ktx2 | [Khronos][Mark_Callow]
```